gcp/app-engine/main.py

```diff
@@ -184,6 +104,42 @@ def post_user(user_email):
 
     return '', 201
 
+@app.route(PUBLIC_API_PREFIX + '/slack/events', methods=['POST'])
+@verify_slack_signature
+def post_slack_events(user_email=None):
```

Comment:
new endpoint to handle events from slack, the main event being processed is on line 128 below for app mentions if you do something like `@Golf Better API`, but i'll probably add slash commands at some point and maybe some other stuff, this is just a start

Reply:
but we should be able to factor out all of this logic (and how the handle_slack_command() parsing the request) to make it easier, faster, and more intuitive. Or maybe using a slash command makes sense, as well as adding autocomplete if we can